### ISSUE TITLE
[GUI] scroll to top in xds config

### ISSUE THREAD

**Issue by bcollard** [kumahq/kuma-gui#3036]
### Description

Would be cool to have a scroll-to-top button in the xds config panel:

![image](https://github.com/user-attachments/assets/5861bf9b-048f-4032-bd2f-bb8293d5816b)

**Comment by johncowen** [comment]
Hey sorry my mistake, this already is a sticky toolbar, so I'm pretty sure the ask is to add the "scroll to top" button, not make the toolbar itself "sticky"

**Comment by bcollard** [comment]
yes, correct. We need a button somewhere on the page to scroll to the very top, instead of manually scrolling up over thousands of lines

**Comment by johncowen** [comment]
Semi related, https://github.com/Kong/kongponents/pull/2570 guess we can just add a button to the secondary-actions once this PR has gone in.

**Comment by johncowen** [comment]
In https://github.com/kumahq/kuma-gui/pull/3711 we turned any "full page codeblocks" into "scrollable codeblocks" instead of using the whole page to scroll (we did this for unrelated performance reasons).

Because of this it means that only the code now scrolls, not the entire page, and due to this this means that all the toolbar buttons plus any other buttons remain in the screen as you scroll the code.

Closing this one, but do let us know if for whatever reason this isn't sufficient 👍
